webui/app.py

```diff
@@ -113,27 +122,33 @@ def stammdaten():
         zustand = request.form.get('zustand')
 
         if len(besucher) == 0:
-            message = 'Neuer Besucher mit ID {} hinzugef&uuml;gt.'.format(besucher_id)
-            print('Adding new user: {}'.format(besucher_id))
-            cursor.execute(new_user_stmt, (besucher_id,
-                                           name, adresse1, plz, adresse2,
-                                           telefon,email,status,coronawarn))
+            if plz in banned_zipcodes:
```

Comment:
You need to define `banned_zipcodes` as an empty list somewhere. Otherwise this throws an error if no zipcode file was provided. "TypeError: argument of type 'NoneType' is not iterable"